libraries/AP_Mount/AP_Mount_SToRM32_serial.cpp

```diff
@@ -71,31 +69,39 @@ void AP_Mount_SToRM32_serial::update()
         // RC radio manual angle control, but with stabilization from the AHRS
         case MAV_MOUNT_MODE_RC_TARGETING: {
             // update targets using pilot's RC inputs
-            MountTarget rc_target {};
-            if (get_rc_rate_target(rc_target)) {
-                update_angle_target_from_rate(rc_target, _angle_rad);
-            } else if (get_rc_angle_target(rc_target)) {
-                _angle_rad = rc_target;
+            if (get_rc_rate_target(mnt_target.rate_rads)) {
+                mnt_target.target_type = MountTargetType::RATE;
+                update_angle_target_from_rate(mnt_target.rate_rads, mnt_target.angle_rad);
+            } else {
```

Comment:
As mentioned above, let's add back in the "if (get_rc_angle_target(rc_target)) {".  It won't make any functional difference but it makes more durable code if we don't assume that we know what will go on in another method.